test/integration_test/applicationbackup_test.go

```diff
@@ -25,13 +26,19 @@ const (
 
 func testApplicationBackup(t *testing.T) {
 	t.Run("applicationBackupRestoreTest", applicationBackupRestoreTest)
+	t.Run("applicationBackupRestorePreExecRuleTest", applicationBackupRestorePreExecRuleTest)
```

Comment:
Don't add `applicationBackupRestore` to the test name, it is already running under the applicationbackup subtest. The function name is fine, since it needs to be unique